validphys2/src/validphys/filters.py

```diff
@@ -168,24 +171,91 @@ def _filter_real_data(filter_path, data):
     return total_data_points, total_cut_data_points
 
 
-def _filter_closure_data(filter_path, data, fakepdf, fakenoise, errorsize):
-    """Filter closure test data."""
+def _filter_closure_data(filter_path, data, fakepdf, fakenoise, filterseed, errorsize):
+    """
+    GENERAL DESCRIPTION:
+
```

Comment:
```suggestion
```

I'd remove these: They don't follow the exsisting style, and it is already clear from the context.